assignment.js

```diff
@@ -32,6 +32,16 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const baseNumber = 2
+let product
+
+for (let i = 0; i <= 12; i++){
+
```

Comment:
@justinbudlow try to avoid having arbitrary blank lines in your code. I generally use blank lines to visually separate logic. However, in this case, I'm not sure it's needed.